### ISSUE TITLE
Icon in Linux

### ISSUE THREAD

**Issue by Johndjk** [KRTirtho/spotube#1315]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Pain

It's difficult to find the app when it's in the background, especially with other apps that don't have the icon either.
Please implement it.
It's essential!

![2024-03-09 13-46-07](https://github.com/KRTirtho/spotube/assets/154085819/325b8031-dfa6-4fa5-83b6-e9a5ef6fce02)

### Suggested solution

Add icon in Linux

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by Peperehobbit01** [comment]
Yeah that's a bugs i have mention here in #1302 , it should have a icon but it dosen't

**Comment by Peperehobbits01** [comment]
Can you close this issues please, this will help maintainers going trhow issues faster. As this bug as been patch it.
